modules/c/pages/gs-install.adoc

```diff
@@ -257,6 +272,12 @@ Be sure to select x64 for 64-bit builds
 Couchbase Lite for C does not have any preferred installation path for the `.dll`.
 It is up to you to determine where best to place it so it is available during execution, though copying to a location on the system path is not recommended on Windows
 
+=== Install Vector Search for Windows
+
+To use the Vector Search extension:
+
+. Download and extract the link:https://www.couchbase.com/downloads/?family=couchbase-lite#LiteExtensions[Vector Search extension].
+. Put the library in your development environment.
```

Comment:
CBL provides CouchbaseLiteVectorSearch.dll. Similar to the instruction to install CBL for windows above, 

```
Couchbase Lite Vector Search does not have any preferred installation path for the `.dll`. It is up to you to determine where best to place it so it is available during execution.
```

Reply:
Done!